Projet-Web-Avance-A4/Developpeur
# feat : connect Developpeur with Microservices

How to test ?

1) Go to localhost:3000 from Developpeur repository
2) Create the database and table in MySql
3) Create an account
4) Check in mysql
